itests/src/test/java/fi/metatavu/edelphi/test/ui/base/AbstractUITest.java

```diff
@@ -537,6 +556,39 @@ protected void takeScreenshot(File file) throws IOException {
       fileOuputStream.close();
     }
   }
+
+  protected void acceptPaytrailPayment(Double expectedAmount) {
+    String paymentAmount = String.format(Locale.GERMAN, " %.2f \u20AC", expectedAmount);
+    waitAndAssertText("#payment-amount", paymentAmount, true, true);
+    
+    waitAndClick("input[value=\"Osuuspankki\"]");
+    waitPresent("*[name='ktunn']");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "*[name='ktunn']" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)